packages/react-native-sdk/docusaurus/docs/reactnative/05-ui-cookbook/10-connection-quality-indicator.mdx

```diff
@@ -3,7 +3,88 @@ id: connection-quality-indicator
 title: Connection Quality Indicator
 ---
 
-:::info
-This guide is still in progress and will be updated soon.
-If you have any questions, please reach out to us on our [website](https://getstream.io/video/#contact).
-:::
+Network conditions are not always perfect so instead of playing guessing games as to why is someone's video so blurry our default [`ParticipantView`](../../ui-components/participants/participant-view/) component comes equipped with connection quality indicator which provides general information about that participants network conditions in four states:
+
+- UNSPECIFIED
+- POOR
+- GOOD
+- EXCELLENT
+
+![Preview of the Connection quality indicator](../assets/06-advanced/10-connection-quality-indicator/connection-quality-indicator.png)
```

Comment:
```suggestion
![Preview of the default Connection quality indicator](../assets/06-advanced/10-connection-quality-indicator/connection-quality-indicator.png)
```

Reply:
can we also add a caption on the image here please that's more clear

https://staging.getstream.io/video/docs/react/ui-cookbook/custom-label/

Reply:
It is done using the ImageShowcase component and which brings the image to the left by default which doesn't look good on the page.
![Uploading Screenshot 2023-07-27 at 2.39.40 PM.png…]()

Reply:
can we use this image instead

https://staging.getstream.io/video/docs/assets/images/network-quality-indicator-a44a4b93a1253e9ca87f030774096385.png

Reply:
ImageShowcase is used when there are multiple images to show

Reply:
If not using ImageShowcase lets add a sentence above saying that this is a preview of the default component please

Reply:
Done